content/doc/developer/handling-requests/deprecated-reflective-access.adoc

```diff
@@ -0,0 +1,19 @@
+---
+title: Deprecated reflective access
+layout: developersection
+---
+
+Before Jenkins 2.272 Stapler would access all fields ignoring visibility restrictions.
+
+Stapler now tries to access a field first and only if it gets access denied then will it force access via 
+javadoc:java.lang.reflect.AccessibleObject#setAccessible-boolean-[setAccessible(true)].
+
+This behaviour is not recommended anymore, and will trigger IllegalReflectiveAccess warnings when running Jenkins
+on Java 11.
+
+If you see a warning message about this then you should link:/participate/report-issue/[report an issue] to the respective plugin.
```

Comment:
Additional explanation to plugin devs might help (use public getters?), but can always be added later.

Reply:
I think we can add later once we see where it gets hit